project/Dependencies.scala

```diff
@@ -38,6 +40,10 @@ object Dependencies {
   val scalapbRuntime      = "com.thesamet.scalapb"       %% "scalapb-runtime"           % scalapb.compiler.Version.scalapbVersion % "protobuf"
   val scalapbRuntimegGrpc = "com.thesamet.scalapb"       %% "scalapb-runtime-grpc"      % scalapb.compiler.Version.scalapbVersion
   val grpcNetty           = "io.grpc"                     % "grpc-netty"                % scalapb.compiler.Version.grpcJavaVersion
+  val nettyBoringSsl      = "io.netty"                    % "netty-tcnative-boringssl-static" % "2.0.8.Final"
```

Comment:
Where do we use those classifiers? Could not find it in code?

Reply:
We are using `nettyBoringSsl` together with `secp256r1` until we find a solution how to use in `sbt` different managed dependencies for `assembly`, `rpm` and `deb` deployments.

Reply:
so other ones should be deleted for now? or you keeping them for the next PR?

Reply:
We are keeping them together with the `Detector`.